src/components/ConsoleLog.tsx

```diff
@@ -0,0 +1,7 @@
+const ConsoleLog = ({ children }: any): any => {
```

Comment:
rename to `Logger` ?

Reply:
Thought of it. But planning to add a proper logging mechanism. This is just a temp solution for now, just to get this started.